lib/bots/versionbot.ts

```diff
@@ -492,22 +497,55 @@ export class VersionBot extends ProcBot {
                     state: 'failure'
                 },
                 method: this.githubApi.repos.createStatus,
-            }).then(() => {
-                // We only complain on opening. Either at that point it'll be fixed, or we
-                // simply don't pass it in future. This stops the PR being spammed on status changes.
-                if (event.cookedEvent.data.action === 'opened') {
+            });
+        }).then(() => {
+            // Check statuses (including Versionist) on the PR.
+            return this.checkStatuses(pr);
+        }).then((checkStatus) => {
+            // If any of them fail (*not* pending), then *if* we haven't already
+            // commented (ie. we previously commented and there's been no commit
+            // since), we ping the author of the PR and whinge at them.
+            if (checkStatus === StatusChecks.Failed) {
+                // Get the last commit. If there have been no comments *since* the
+                // date of that commit, then we know that we can safely post a
+                // comment telling the author of a failure. If there has, then
+                // it's implicit that we made a comment.
+                const lastCommitTimestamp = lastCommit.commit.committer.date;
+                return this.githubCall({
+                    data: {
+                        owner,
+                        repo: name,
+                        number: pr.number,
+                    },
+                    method: this.githubApi.issues.getComments
+                }).then((comments: GithubApiTypes.Comment[]) => {
+                    // Check for any comments that come *after* the last commit
+                    // timestamp and is a Bot.
+                    let foundBotComment = false;
+                    for (let comment of comments) {
+                        if ((Date.parse(lastCommitTimestamp) < Date.parse(comment.created_at)) &&
+                            (comment.user.type === 'Bot')) {
```

Comment:
In case we end up with multiple ProcBots, all commenting on PRs about different things, would it be possible to tighten this check to only check for comments by VersionBot?